roadmap.md

```diff
@@ -25,49 +18,87 @@ state.
 
 ### Work in progress (WIP)
 
-1. **Make artifact validation more robust**
+1. **Rewrite artifact promoter**
 
-   Outcome: Provide a fast and error-resistant workflow for artifact promotion
-   compliant with SLSA.
+   Outcome: Robust and fast artifact promotion tooling.
 
-   Enhancement: was https://github.com/kubernetes/enhancements/issues/3027
+   Enhancement: _none_
 
    Project board: https://github.com/orgs/kubernetes/projects/171
 
-1. **Making deb and rpm package builds more robust**
+1. **Rework supported platforms**
 
-   Outcome: Provide a fast and error-resistant workflow for building and
-   publishing packages to the community-owned infrastructure.
+   Outcome: Which platforms Kubernetes supports reflected in documentation and tooling.
 
-   Enhancement: https://github.com/kubernetes/enhancements/issues/1731
+   Enhancement: **TBD** ([Supported Platform Tiers](https://docs.google.com/document/d/1QnT2LJ1dNYFyvOCt_X6F8ry0KgAg8H2MjoZnt5Xgtog))
 
-   Project board: https://github.com/orgs/kubernetes/projects/137
+   Project board: _none_
 
-1. **Enable other Kubernetes subprojects to use our packages infrastructure**
+1. **Improve SIG Release contributor ladder & sustainability**
+   Outcome: Up-to-date documentation and project boards for new and long term contributors.
```

Comment:
I think its worth calling out exactly where you need documentation updated, since new contributors could be willing to sign up to help with that as a mechanism for learning about the roles and responsibilities of the different subprojects (e.g. Release Managers, Release Team, etc).

Reply:
@kubernetes/release-managers @xmudrii do you have further thoughts on that?

Reply:
@natalisucks I think we don't know yet and have to find out.

Reply:
As @saschagrunert said, we don't know yet and have to find out. These efforts have to be kicked off by the SIG leads and Release Managers as they have the access to the infrastructure, and then eventually delegate tasks to other contributors.